test/clientside/independent-overrides.js

```diff
@@ -10,8 +10,8 @@ var foober =  proxyquire('../fixtures/foo', { './bar': barber });
 var foo    =  proxyquire('../fixtures/foo', { './bar': { } });
 
 test('\noverriding bar.bar for foober but not for foo', function (t) {
-  t.equal(window.foostats.fooRequires(), 3, 'foo is required three times since one for each test and one for require detective')
```

Comment:
This is the only test I change, because as I mentioned you were incorrectly expecting the module to be required 3 times, 1 for the magic require, and 2 for the require calls.